xpack_test.go

```diff
@@ -0,0 +1,146 @@
+package elastic
+
+import (
+	"context"
+	"testing"
+)
+
+func TestWatchWorkFlow(t *testing.T) {
```

Comment:
Can we use an `Xpack` or `XPack` prefix for the tests, please? That makes testing the X-Pack features easier as we can run `go test -run=XPack` and be sure that we hit all X-Pack-related tests.